composer.json

```diff
@@ -23,7 +23,7 @@
         "doctrine/doctrine-bundle": "^1.6",
         "doctrine/migrations": "^1.4",
         "doctrine/doctrine-migrations-bundle": "^1.1",
-        "twig/extensions": "^1.3",
+        "twig/extensions": "^1.5.1",
```

Comment:
@bartdc I think the lock file is still missing in this pr

Reply:
This file didn't change, how can I push it?

Reply:
@bartdc the file should have changed once you've run `composer install`

Reply:
@bartdc have you been able to look at this?

Reply:
The composer.json file didn't change after composer install. Any tips?

Reply:
How did you update the package? You should either 
- change composer file and do `composer update twig/extensions`
- or just do `composer require twig/extensions:1.5.1`

Reply:
@bartdc the `composer.json` file shouldn't change, it's the `composer.lock` file that should change.